packages/ui-top-nav-bar/src/TopNavBar/TopNavBarLayout/SmallViewportLayout/props.ts

```diff
@@ -113,6 +113,11 @@ type SmallViewportLayoutOwnProps = {
    * Callback fired when an item is selected in the dropdown menu
    */
   onDropdownMenuSelect?: DrilldownProps['onSelect']
+
+  /**
+   * Stay away...
+   */
```

Comment:
Please add a proper description. We should also mention that this is a deprecated prop that people should not use

Reply:
done